c2corg_ui/static/js/apiservice.js

```diff
@@ -147,8 +151,12 @@ app.Api.prototype.unassociateDocument = function(parentId, childId) {
   };
 
   var promise = this.deleteJson_('/associations', data);
-  promise.catch(function() {
-    var msg = alerts.gettext('Failed to unassociate document');
+  promise.catch(function(res) {
+    var error = '';
+    for (var i = 0; i < res['data']['errors'].length; i++) {
+      error = error + ' ' + res['data']['errors'][i]['description'];
+    }
```

Comment:
Ehm, 20 lines below in function `unassociateDocument` you have exactly the same code (but for 2 characters). You saw that?